docs/references/expo/use-sso.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: '`useSSO()`'
+description: Clerk's useSSO() hook is used to create a new SSO flow.
+---
+
+The `useSSO()` hook is used to create a new SSO flow. It can be used in both web and native apps.
+
+## Parameters
+
+<Properties>
+  - `strategy`
+
+  - [`OAuthStrategy`](/docs/references/javascript/types/sso#o-auth-strategy)
+
+  - [`EnterpriseSSOStrategy`](/docs/references/javascript/types/sso#enterprise-sso-strategy)
+
+  - `strategy?`
+
+    - `string`
+
+    Select the SSO strategy. The `enterprise_sso` value depends on the object's `identifier` value. Possible `strategy` values are:
+
+    - `oauth_<provider>`: The user will be authenticated with their social sign-in account. [See available social providers](/docs/references/javascript/types/oauth#o-auth-provider).
+    - `enterprise_sso`: The user will be authenticated either through SAML or OIDC, depending on the configuration of the [enterprise connection](/docs/authentication/enterprise-connections/overview) matching the identifier.
+
+    ---
+
+    - `redirectUrl?`
+    - `string`
+
+    The URL or path to redirect to after the SSO flow is complete.
+
+    ---
+
+    - `unsafeMetadata?`
+    - [`SignUpUnsafeMetadata`](/docs/references/javascript/types/metadata#sign-up-unsafe-metadata)
+
+    Unsafe metadata to be passed to the SSO provider.
+</Properties>
+
+## Returns
+
+The `useSSO()` hook returns the `useSSOFlow()` method, which you can use to initiate the SSO flow.
+
+The `startSSOFlow()` method has the following signature:
+
+```ts
+const startSSOFlow: (startSSOFlowParams?: StartSSOFlowParams) => Promise<StartSSOFlowReturnType>
+```
+
+It accepts the following parameters (`StartSSOFlowParams`):
+
+<Properties>
+  - `redirectUrl?`
+  - `string`
+
+  The URL or path to redirect to after the OAuth flow is complete.
+
+  ---
+
+  - `unsafeMetadata?`
+  - [`SignUpUnsafeMetadata`](/docs/references/javascript/types/metadata#sign-up-unsafe-metadata)
+
+  Unsafe metadata to be passed to the OAuth provider.
+
+  ---
+
+  - `identifier?`
+  - string
+
+  The [identifier](/docs/authentication/configuration/sign-up-sign-in-options#identifier) of the user. Required for `enterprise_sso` strategy.
+</Properties>
+
+## How to use `useSSO()`
+
+<Tabs items={["With OAuth", "With SAML"]}>
+  <Tab>
+    The following example demonstrates how to create a custom SSO sign-in flow for [Google accounts](/docs/authentication/social-connections/google).
```

Comment:
If this is a custom sign-in flow, do we still want to link to this page?

Reply:
Which page? Do you mean /docs/authentication/social-connections/google?

Reply:
Yes! Would it make sense?